server/src/instant/admin/routes.clj

```diff
@@ -477,6 +480,28 @@
                   files)]
     (response/ok {:data data})))
 
+(defn presence-get [req]
+  (let [{app-id :app_id} (req->admin-token! req)
+        _room-type (ex/get-param! req [:params "room-type"] string-util/coerce-non-blank-str)
```

Comment:
Could use a comment about why we require this field but don't use it.

Reply:
_makes salute_ on it!